.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Good job so far :+1:  But I noticed that your linters are not correctly set up:cry: Files stylelintrc.json, .hintrc are missing in your setup. Please follow this [Linters HTML and CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) and don't forget to ignore `node_modules` as required for this project. Linters are very important because they will help to become a better developer by fixing code quickly and having your code syntax correct. :+1: 

![Screenshot from 2022-10-25 09-17-56](https://user-images.githubusercontent.com/6615394/197735431-f4df3e1a-1751-4b11-8841-ad2d253a692d.png)

Reply:
I think i have solved all issues correctly